filebeat/module/nginx/access/test/test-with-host.log-expected.json

```diff
@@ -2,12 +2,19 @@
     {
         "@timestamp": "2016-12-07T10:05:07.000Z",
         "destination.domain": "example.com",
+        "event.category": [
+            "web"
+        ],
         "event.dataset": "nginx.access",
+        "event.kind": "event",
         "event.module": "nginx",
+        "event.outcome": "success",
         "event.timezone": "-02:00",
+        "event.type": [
+            "access"
+        ],
         "fileset.name": "access",
-        "http.request.method": "GET",
-        "http.request.referrer": "-",
+        "http.request.method": "get",
```

Comment:
This looks like a breaking change? `GET` is now lowercased, also referrer is now missing

Reply:
I'm fine with marking this as a breaking change.

Some background.
http.request.referrer is an ECS field and docs say "The field value must be normalized to lowercase for querying."  So that is why the change to lowercase.

http.request.referrer isn't missing, the GROK pattern was changed so it is only filled in when nginx logs a referrer an actual referrer.  "-" is the same as not present.